.coderabbit.yaml

```
@@ -0,0 +1,15 @@
+# yaml-language-server: $schema=https://coderabbit.ai/integrations/schema.v2.json
```

yamllint

[error] wrong new line character: expected \n

(new-lines)